sourcecode/hub/app/mail/ResetPasswordEmail.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class ResetPasswordEmail extends Mailable
+{
+    use Queueable;
+    use SerializesModels;
+
+    public string $resetLink;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param string $resetLink
+     * @return void
+     */
+    public function __construct($resetLink)
+    {
+        $this->resetLink = $resetLink;
+    }
+
+    /**
+     * Build the message.
```

Comment:
Unnecessary comment. A real type should also be added to this method to ensure it always behaves correctly.